src/gui/guiFormSpecMenu.h

```diff
@@ -472,7 +472,7 @@ class GUIFormSpecMenu : public GUIModalMenu
 private:
 	IFormSource        *m_form_src;
 	TextDest           *m_text_dst;
-	u32                 m_formspec_version = 0;
```

Comment:
This change does not matter, because the version was never increased. 0 behaves the same as 1.